tsdb/mapper.go

```diff
@@ -428,7 +433,7 @@ func (lm *SelectMapper) nextChunkAgg() (interface{}, error) {
 			// changes to the mapper functions, which can come later.
 			// Prime the buffers.
 			for i := 0; i < len(tsc.cursors); i++ {
-				k, v := tsc.cursors[i].SeekTo(tmin)
+				k, v := tsc.cursors[i].SeekTo(qmin)
```

Comment:
no, this was indeed part of the bug.  Before it would seek to `tmin`, and it could find values that were `before` the time range asked for, so it would exit the cursor prematurely if that happened.